requirements/edx/github.txt

```diff
@@ -47,7 +47,6 @@
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
 git+https://github.com/edx/django-wiki.git@v0.0.5#egg=django-wiki==0.0.5
 git+https://github.com/edx/django-openid-auth.git@0.8#egg=django-openid-auth==0.8
--e git+https://github.com/edx/django-rest-framework-oauth.git@f0b503fda8c254a38f97fef802ded4f5fe367f7a#egg=djangorestframework-oauth==1.0.1
```

Comment:
Looks like you're right @nedbat. I experimented locally with a virtualenv by installing djangorestframework-oauth 1.0.1 as editable, then performing a regular install of djangorestframework-oauth 1.1.0. pip uninstalls the editable 1.0.1 before installing 1.1.0. I'll remove the explicit uninstall.